spec/plane_spec.rb

```diff
@@ -17,14 +18,33 @@
 
 describe Plane do
 
-  xit 'is flying when created'
+subject {Plane.new}
+let(:airport) { Airport.new }
 
-  xit 'can land'
+  before(:each) do
+      allow(airport).to receive(:can_land).and_return true
+      allow(airport).to receive(:can_take_off).and_return true
+    end
```

Comment:
Lint/BlockAlignment: `end` at 27, 4 is not aligned with `before(:each) do` at 24, 2